web-local/src/routes/(application)/dashboard/[name]/+page.svelte

```diff
@@ -97,7 +98,9 @@
       {#key metricViewName}
         <DashboardStateProvider {metricViewName}>
```

Comment:
The `DashboardStateProvider` has a spinner that should be colored with the `secondary` color, if possible

Reply:
Thoughts on this? I see the `dashboard-theme-boundary` is set deeper in the component tree than the state provider

Reply:
The problem here is we wont have the `validSpec` locally so we cant really use themes.
(Sorry I thought I hit comment)